### PR TITLE
reg-query: add page

### PR DESCRIPTION
**Pull request by pxgamer**
Part of #1908

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

**Comment by pxgamer** [comment]
If they're not already done, I will create the rest when I'm back on Monday. :relaxed:

**Comment by sbrl** [comment]
Awesome, @pxgamer! Thanks for all your hard work  😺

File: pages/windows/reg-query.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+- Display all values of a key:
+
+`reg query {{key_name}}`
+
+- Display a specific value of a key:
+
+`reg query {{key_name}} /v {{value}}`
+
+- Display a specific value of a key and recurse into sub keys:
```
Comment (agnivade) [inline pages/windows/reg-query.md]: You don't specify a value here, yet the description says "a specific value of a key". Typo ? Does this sound better - `Display all values of a key and its sub keys` ?

Comment (pxgamer) [reply pages/windows/reg-query.md]: Yes. Sorry, that is a better description. 👍

**Pull request merged**